packages/Manager/src/main/resources/application.yml

```diff
@@ -21,14 +21,13 @@ spring:
         spring:
           json.trusted.packages: nz.govt.eop.messages
           deserializer.value.delegate.class: org.springframework.kafka.support.serializer.JsonDeserializer
-          spring.json.use.type.headers: false
+          json.use.type.headers: false
```

Comment:
Few places where it got double spring'ed and were warning messages about
`spring.spring.json.use.type.headers` being an unknown option